src/main/scala/rifs/business/tables/OpportunityTables.scala

```diff
@@ -110,6 +110,15 @@ class OpportunityTables @Inject()(val dbConfigProvider: DatabaseConfigProvider,
   lazy val sectionC = Compiled(sectionQ _)
 
   override def saveSectionDescription(id: OpportunityId, sectionNo: Int, description: Option[String]): Future[Int] = db.run {
-    sectionQ(id, sectionNo).map(r=> r.text).update(description)
+    sectionQ(id, sectionNo).map(r => r.text).update(description)
+  }
+
+  override def reset(): Future[Unit] = {
```

Comment:
Well, smaller number are more human-friendly, but that also encourages tests being reliant on them.